tests/t259_arg_struct_script.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+from runtest import TestBase
+import subprocess as sp
+
+class TestCase(TestBase):
+    def __init__(self):
+        TestBase.__init__(self, 'struct', """
+uftrace_begin(ctx)
+  record  : False
+  version : v0.10-17-g8d1b ( x86_64 dwarf python luajit tui perf sched dynamic )
+  cmds    : t-struct
+
+11332966.196463691  50529: [entry] main(4006ef) depth: 0
+11332966.196464540  50529: [entry] foo(40068f) depth: 1
+  args[0] <class 'str'>: struct: Option{}
+  args[1] <class 'str'>: struct: StringRef{}
+  args[2] <class 'int'>: 44
+  args[3] <class 'int'>: 55
+  args[4] <class 'int'>: 66
+11332966.196670289  50529: [exit ] foo(40068f) depth: 1
+11332966.196671664  50529: [exit ] main(4006ef) depth: 0
+  retval  <class 'int'>: 0
+
+uftrace_end()
+""", cflags='-g', sort='dump')
+
+    def build(self, name, cflags='', ldflags=''):
+        if not 'dwarf' in self.feature:
+            return TestBase.TEST_SKIP
```

Comment:
I think it should also skip if 'python' is not in the feature.

Reply:
done